File-Formats/PDF-to-image/Convert-PDF-to-image-using-PDFium.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Convert PDF to image using PDFium | Syncfusion
+description: Learn about Converting PDF pages into images using Syncfusion PdfToImageConverter with the help of PDFium.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+
+# Convert PDF to image using PDFium
+
+PDFium is used in Google Chrome for rendering PDF files. It provides accurate and robust PDF rendering. It is the recommended PDF rendering engine. 
+
+N> PDFium rendering is not supported in Windows XP operating system.
+
+## How to run PDFium in a restricted access environment
```

Comment:
add one more topic as "How do PdfIum works with PdfToImageConverter, in below referring below Ug
https://help.syncfusion.com/windowsforms/pdf-viewer/pdf-rendering-engines#how-pdfium-works-with-syncfusions-pdf-viewer
Note add NuGet extracting behavior

Reply:
I have modified as mentioned